app/browser/share.js

```diff
@@ -19,6 +19,11 @@ const templateUrls = {
   reddit: 'https://reddit.com/submit?url={url}&title={title}'
 }
 
+// Sites that are not currently in the File > Share context menu
+const excludedShareSites = [
```

Comment:
Do we need this rather than just remove the _digg_ item from `templateUrls`. From a quick search I could not find any other usage of the `templateUrls` export from _share.js_.

Reply:
@petemill good thought, I did a search and it looks fine to remove. I've pushed an update removing the `excludedShareSites` addition. I left removing Digg from `templateUrls` a separate commit for ease of search should we want to add it back at any point.